### PR TITLE
229 show transaction types in cards

### PR DESCRIPTION
**Pull request by Rashida1akter**

### CODE REVIEW

**Comment by Rashida1akter** [comment]
MicrosoftTeams-image

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { API_HOST } from "../../constants";
+import "./partner_balances.css";
+export interface ITransactionType {
+  userId: string;
+  currency: string;
+  credentials: any;
+}
+const TransactionType = (props: ITransactionType) => {
+  const [loading, setLoading] = useState(false);
+  const [balance, setBalance] = useState<any>();
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:13]: is it really a balance ? I think it should be transactionTypes

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <>
+      {" "}
+      <h2>Transaction Types : {props.userId}</h2>
+      {loading && <h1>Loading</h1>}
+      <div className="wrapper">
+        {!loading &&
+          balance?.map((record: { id: string; label: string }) => (
+            <div className="card">
+              <p>
+                <strong>{Object.keys(record)[0]}</strong> : {record.id}
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:45]: {Object.keys(record)[0]} ?? should be id

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+              </p>
+              <p>  <strong>{Object.keys(record)[1]}</strong> : {record.label}</p>
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:47]: {Object.keys(record)[1]} ?? should be label

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      );
+      if (fetchBalance.data) {
+        const items = fetchBalance.data;
+        setBalance(items);
+      }
+      setLoading(false);
+    };
+    fetchData();
+  }, [props.userId, props.currency]);
+  console.log(balance);
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:33]: remove console.log

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  //   c58d2999-42df-401d-9eef-32a584232370
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:34]: remove id

File: src/lib/PartnerBalances/TransactionTypes.tsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+const TransactionType = (props: ITransactionType) => {
+  const [loading, setLoading] = useState(false);
+  const [balance, setBalance] = useState<any>();
+  useEffect(() => {
+    const fetchData = async () => {
+      setLoading(true);
+      const fetchBalance = await axios.post(
+        `${API_HOST}/tenant/${props.credentials.application_id}/get-transaction-type-autocomplete?filter[userId]=${props.userId}`,
+
+        // /api//tenant/:tenantId/get-transaction-type-autocomplete
```
Comment (ishaan-puniani) [inline src/lib/PartnerBalances/TransactionTypes.tsx:20]: remove commented code

**Review by ishaan-puniani — CHANGES_REQUESTED** [review]

**Pull request merged**